services/storage/requirements/_test.in

```diff
@@ -5,11 +5,14 @@
 --constraint ../../../requirements/constraints.txt
 --constraint _base.txt
 
+boto3==1.21.21

```

Comment:
Why this version?
Where is the documentation for this version?
I only find latest in https://boto3.amazonaws.com/v1/documentation/api/latest/index.html

Reply:
I'll try to update it to see if now I can unfix it